adoc/MAIN.release-notes.adoc

```diff
@@ -36,7 +36,11 @@ The official website for the {tf} 0.12 upgrade is https://www.terraform.io/upgra
 
 === Add CLI etcdctl
 
-You can now install `etcdctl`, the CLI to provide access to etcd container from the management host.
+You can now install `etcdctl`, the CLI to provide access to etcd cluster from the management host to backup and restore cluster data. For detailed instructions please see link: https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-deployment/#_backup_etcd_cluster_data[the Deployment Guide].
+
+=== Add Discover and Recovery 
+
+You can now install `velero`, the CLI to provide access to backup cluster data to S3 storage or S3-compatible storage and recover/migrate to a different cluster. For detailed instructions please see link: https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-deployment/#_disaster_Recovery[the Deployment Guide].
```

Comment:
I cannot check the link is correct or not, but is it `#_disaster_Recovery` or `#_disaster_recovery` or ...?

Reply:
the doc is not merge/publish yet. I just created the link following caption convention to link.

Reply:
You can typically check the state of `master` on https://susedoc.github.io/doc-caasp/master/single-html/caasp-admin/#_disaster_recovery once it's merged it takes a few minutes to be rebuild by the CI.

Then you just need to replace the first bit with the parts for the final URL.
`https://susedoc.github.io/doc-caasp/master/` with
`https://documentation.suse.com/suse-caasp/4.1/`. If the URL works on github.io then it should work in the final published document.